corehq/apps/app_manager/tests/test_profile.py

```diff
@@ -1,23 +1,28 @@
 # coding: utf-8
-from django.test import TestCase
+from django.test import SimpleTestCase
 from corehq.apps.app_manager.commcare_settings import SETTINGS_LOOKUP, SETTINGS
 from corehq.apps.app_manager.models import Application
 from corehq.apps.app_manager.tests.util import TestFileMixin
 import xml.etree.ElementTree as ET
 
 from corehq.apps.builds.models import BuildSpec
+from corehq import toggles
+from toggle.shortcuts import update_toggle_cache
 
 
-class ProfileTest(TestCase, TestFileMixin):
+class ProfileTest(SimpleTestCase, TestFileMixin):
     file_path = ('data',)
 
     def setUp(self):
         self.app = Application(build_spec=BuildSpec(
             version='2.7.0'
             ),
-            name=u"TÉST ÁPP"
+            name=u"TÉST ÁPP",
+            domain="potter"
         )
 
+        update_toggle_cache(toggles.CUSTOM_PROPERTIES.slug, 'potter', True, 'domain')
```

Comment:
You should use the `corehq.toggle.NAMESPACE_DOMAIN` constant to make it obvious what's going on.

Also I think it would be prudent to add a `tearDown` method that clears the toggle cache in case it messes with other tests in the future.